src/protocol.h

```diff
@@ -265,6 +265,12 @@ extern const char* WTXIDRELAY;
  * txreconciliation, as described by BIP 330.
  */
 extern const char* SENDTXRCNCL;
+/**
+ * Contains a 4-byte local reconciliation set size and 4-byte q-coefficient
```

Comment:
I think the pair of values returned by `InitiateReconciliationRequest` is `std::optional<std::pair<uint16_t, uint16_t>>` so should say 2-byte local reconciliation set size and 2-byte q-coefficient.